### PR TITLE
changes to get trusty working

### PR DESCRIPTION
**Pull request by paulczar**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -145,6 +145,10 @@ To deploy your `allinone` environment via `Vagrant` simply run:
 $ bin/run_vagrant up allinone
 ```
 
+Note: The default OS for ursula is Ubuntu Precise, if you want Trusty set
```
Comment (dlundquist) [inline README.md]: Can we set default to Trusty?

Comment (paulczar) [inline README.md]: Sure,  that note only applies to `Vagrant` though ofc.   otherwise the default OS is the one you're deploying to. :)

**Comment by j2sol** [comment]
I think this looks fine.

**Comment by dlundquist** [comment]
+1

**Pull request merged**